AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginTests/Sync/InitialSync/InitialSyncOperationTests.swift

```diff
@@ -83,11 +108,33 @@ class InitialSyncOperationTests: XCTestCase {
             api: apiPlugin,
             reconciliationQueue: reconciliationQueue,
             storageAdapter: storageAdapter,
-            dataStoreConfiguration: .default) { _ in }
+            dataStoreConfiguration: .default)
+
+        let syncStartedReceived = expectation(description: "Sync started received, sync operation started")
+        let syncCompletionReceived = expectation(description: "Sync completion received, sync operation is complete")
+        let finishedReceived = expectation(description: "InitialSyncOperation finishe offering items")
+        let sink = operation
+        .publisher
+        .sink(receiveCompletion: { _ in
+            syncCompletionReceived.fulfill()
+        }, receiveValue: { value in
+            switch value {
+            case .started(modelType: let modelType, syncType: let syncType):
+                XCTAssertEqual(modelType.modelName, "MockSynced")
+                XCTAssertEqual(syncType, SyncType.fullSync)
+                syncStartedReceived.fulfill()
+            case .finished(modelType: let modelType):
+                XCTAssertEqual(modelType.modelName, "MockSynced")
+                finishedReceived.fulfill()
+            default:
+                break
+            }
+        })
 
         operation.main()
 
-        wait(for: [apiWasQueried], timeout: 1.0)
+        waitForExpectations(timeout: 1, handler: nil)
```

Comment:
`handler` has a default value so you can omit: `waitForExpectations(timeout: 1)`

Reply:
updated

Reply:
updated